### PR TITLE
Feature#14 Internationalization 

### PR DESCRIPTION
**Pull request by cortezroberto**
# Description

 - We've implemented react-native-i18n library to use internationalization instead of static text .
 
 Examples:
 - <'Text>I agree with the Terms and Privacy Policy. *</'Text>
 - <'Text>{I18n.t('agreeTerms')}</'Text>

### CODE REVIEW

**Review by DiegoSpielzeug — APPROVED** [review]
LGTM!

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by jkzasori — APPROVED** [review]

**Review by jkzasori — COMMENTED** [review]
LGTM!

File: FlightReservation/src/screens/SignUp.js [hunk 1]
```diff
@@ -65,40 +75,42 @@ const SignUp = function ({navigation}) {
...
                 <View style={style.lower_content_text}>
                   <Text style={style.alreadyTxt}>
-                    Already have an account?
+                    <Span text="alreadyAccount" />
                     <Text
                       style={style.account_link}
                       onPress={() => {
                         navigation.navigate('LogIn');
                       }}>
                       {' '}
```
Comment (jkzasori) [inline FlightReservation/src/screens/SignUp.js:112]: You could delete this line, but it's okay.

**Review by DemisRincon — APPROVED** [review]

**Pull request merged**